customer/src/main/java/no/unit/nva/customer/WriteControlledVocabularyHandler.java

```diff
@@ -0,0 +1,33 @@
+package no.unit.nva.customer;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.util.UUID;
+import no.unit.nva.customer.model.CustomerDto;
+import no.unit.nva.customer.model.interfaces.VocabularySettingsList;
+import no.unit.nva.customer.service.CustomerService;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+
+public abstract class WriteControlledVocabularyHandler
```

Comment:
Contains the common logic of CreateVocabularySettings and UpdateVocabularySettings